guide/de/ch_txns.xml

```diff
@@ -295,15 +295,15 @@
           Schiebefenster eine bestimmte Anzahl von Monaten vor dem laufenden Monat genommen wird.
           Dies wird unter
           <menuchoice>
-            <guimenu>Bearbeiten</guimenu><guisubmenu>Einstellungen</guisubmenu><guisubmenu>Datum und
+            <guimenu>Bearbeiten</guimenu><guisubmenu>Einstellungen</guisubmenu><guisubmenu>Zahl, Datum, 
             Zeit</guisubmenu><guilabel>Datum vervollständigen</guilabel>
           </menuchoice>
```

Comment:
The elements for listing the menu items are `<guimenu>`, `<guimenuitem>` and `<guisubmenu>`. So the menu item is as follows: `<guimenuitem>Einstellungen</guimenuitem>`.

Reply:
Fixed in the merge commit.